docs/getting-started/products/ylockers/overview.md

```diff
@@ -1,8 +1,111 @@
 # Overview
-yLockers is a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
 
-To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1. 
+:::info
+
+The new Ylockers ecosystem is rolling out on May 9th, 2024 with the launch of the *YearnBoostedStaker* (YBS) contract and a new V3 Auto-Compounder vault. These new contracts will be available for yPRISMA initially and then be rolled out for yCRV a month later on June 6th. Before June 6th, yCRV will continue to operate with the existing st-yCRV and lp-yCRV V2 vaults.
+
+:::
+
+yLockers are a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
+
+To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1.
 
 The benefit of doing this is to provide the end user with a fully transferrable and liquid token that can still receive a share of governance benefits like yield or voting power depending on the protocol. No longer is there a need to lock up your asset value for up to 4 full years!
 
 > **⚠️ Important**: Liquid locker tokens are not redeemable for the underlying locked tokens. This is not possible. But because they are liquid, they can be traded on decentralized exchanges, and bought and sold at market value.
+
+## yLocker Products
+
+* [yPrisma](../ylockers/yprisma/overview)
+* [yCRV](../ylockers/ycrv/overview)
+
+## Earning Yield with yLockers
+
+Within the yLockers ecosystem, there are two distinct ways to earn yield:
+
+1. Deposit your yLocker tokens into YBS (earn stablecoins)
+2. Deposit your yLocker tokens into the V3 Auto-Compounder (earn yLocker tokens)
+
+___
+
+### Option 1: Deposit yLocker Tokens into YBS
+
+Each week, Yearn's yLockers earn revenue from protocol fees and bribes. This revenue is converted into ecosystem stablecoins (mkUSD for yPrisma or crvUSD for yCRV) and distributed to yLocker stakers at the start of the week via the `YearnBoostedStaker`(YBS), a new contract which was launched in Spring 2024 following the passing of [YIP-77](https://snapshot.org/#/veyfi.eth/proposal/0xe79fb2ef4f21ef1e9cc30dd1522c9751c74b631c4782bccbbeb25185d4ddae1d).
```

Comment:
Maybe I'm being a bit sensitive, but there has been a push recently among many protocols to stop calling these `bribes`. Something like `vote rewards` or `vote incentives` is probably a better practice

Reply:
If you could also apply this to the other places `bribes` are mentioned as well :)

Reply:
I agree wholeheartedly. I like `vote incentives` personally. Will change in these docs for this pull request and then update throughout the docs in another pull-request.

Reply:
yBribe might be a hard nut to crack in this regard though 😄

Reply:
lol yeah I thought of that, fortunately (at least in this sense, lol) we're killing yBribe since it's not really used

Reply:
updated everything to `Vote Incentives` except for in yBribe. Will deal with that later.